src/ContactsTabManager.php

```diff
@@ -156,7 +150,7 @@ public function getTabs(UserInterface $contact) {
   /**
    * {@inheritdoc}
    */
-  public function getBlocks(ContactTabInterface $tab, UserInterface $contact = NULL, $verify = TRUE) {
+  public function getBlocks(ContactTabInterface $tab, UserInterface $contact = NULL) {
     $blocks = $tab->getBlockPlugins();
     if (empty($blocks)) {
```

Comment:
Think we still want this to be an `if (!isset($blocks))` with the default being NULL, so we can tell the difference between no blocks and blocks not gathered?